taipy/core/_entity/_migrate/_utils.py

```diff
@@ -241,6 +255,9 @@ def __migrate_global_config(config: Dict):
     except KeyError:
         pass
 
+    if "core_version" not in config["CORE"]:
```

Comment:
```suggestion
    if "core_version" not in config["CORE"] or config["CORE"]["core_version"] != version("taipy-core"):
```

We need to update the version if it is outdated as well

Reply:
Good catch.